src/procedures/__tests__/EnableDividendManagers.ts

```diff
@@ -0,0 +1,117 @@
+import { ImportMock, MockManager } from 'ts-mock-imports';
+import { SinonStub, stub, spy, restore } from 'sinon';
+import BigNumber from 'bignumber.js';
+import * as contractWrappersModule from '@polymathnetwork/contract-wrappers';
+import * as contextModule from '../../Context';
+import * as wrappersModule from '../../PolymathBase';
+import * as tokenFactoryModule from '../../testUtils/MockedTokenFactoryObject';
+import { EnableDividendManagers } from '../../procedures/EnableDividendManagers';
+import { Procedure } from '~/procedures/Procedure';
+import { PolymathError } from '~/PolymathError';
+import { ErrorCode, PolyTransactionTag } from '~/types';
+
+const params1 = {
+  symbol: 'TEST1',
+  name: 'Test Token 1',
+  amount: new BigNumber(1),
+  checkpointIndex: 1,
+  maturityDate: new Date(2030, 1),
+  expiryDate: new Date(2031, 1),
+  address: '0x4444444444444444444444444444444444444444',
+};
+
+describe('EnableDividendManagers', () => {
+  let target: EnableDividendManagers;
+  let contextMock: MockManager<contextModule.Context>;
+  let wrappersMock: MockManager<wrappersModule.PolymathBase>;
+  let tokenFactoryMock: MockManager<tokenFactoryModule.MockedTokenFactoryObject>;
+  let etherDividendsMock: MockManager<contractWrappersModule.EtherDividendCheckpoint_3_0_0>;
+  let securityTokenMock: MockManager<contractWrappersModule.SecurityToken_3_0_0>;
+  let tokenFactoryMockStub: SinonStub<any, any>;
+  let getAttachedModulesMockStub: SinonStub<any, any>;
+
+  beforeAll(() => {
+    // Mock the context, wrappers, and tokenFactory to test CreateEtherDividendDistribution
+    contextMock = ImportMock.mockClass(contextModule, 'Context');
+    wrappersMock = ImportMock.mockClass(wrappersModule, 'PolymathBase');
+    tokenFactoryMock = ImportMock.mockClass(tokenFactoryModule, 'MockedTokenFactoryObject');
+    contextMock.set('contractWrappers', wrappersMock.getMockInstance());
+    wrappersMock.set('tokenFactory', tokenFactoryMock.getMockInstance());
+
+    securityTokenMock = ImportMock.mockClass(contractWrappersModule, 'SecurityToken_3_0_0');
+    securityTokenMock.mock('address', Promise.resolve(params1.address));
+
+    etherDividendsMock = ImportMock.mockClass(
+      contractWrappersModule,
+      'EtherDividendCheckpoint_3_0_0'
+    );
+    getAttachedModulesMockStub = wrappersMock.mock(
+      'getModuleFactoryAddress',
+      Promise.resolve(params1.address)
+    );
+    tokenFactoryMockStub = tokenFactoryMock.mock(
```

Comment:
This stub doesn't seem to be used anywhere, so I don't see the need to assign it to a variable